client/src/panels/welcome_mat/welcome_mat.js

```diff
@@ -158,8 +158,8 @@ const WelcomeMat = (props) => {
     latest_hist_spend_data,
     oldest_hist_spend_data,
   } = calcs;
-
-  const actual_hist_years_apart = _.parseInt( _.split(latest_hist_spend_data.year, '-') ) - _.parseInt( _.split(oldest_hist_spend_data.year, '-') ) + 1;
+  const current_year = _.parseInt( _.split( _.last(actual_history_years), '-') ) + 1;
```

Comment:
Can get the value of `current_year` directly from `_.parseInt( run_template('{{pa_in_year_short_first}}') )`.